lib/beaker-puppet/install_utils/module_utils.rb

```diff
@@ -114,13 +114,16 @@ def install_puppet_module_via_pmt( opts = {} )
         def copy_module_to(one_or_more_hosts, opts = {})
           block_on one_or_more_hosts do |host|
             opts = {:source => './',
-                    :target_module_path => host['distmoduledir'],
```

Comment:
In this version of code we had this default, which is lost (to the autodetect)

Reply:
I considered autodetect to be better, but I don't feel strongly about this.

Reply:
I have a commit locally that reverts to the old behavior but it also includes the path checking code which I haven't had time to test yet.